python-test2.py

```diff
@@ -0,0 +1,75 @@
+import random
+
+class InventoryItem:
+    def __init__(self, name, quantity, price):
+        self.name = name
+        self.quantity = quantity
+        self.price = price
+
+    def update_price(self, new_price):
+        if new_price > 0:
+            self.price = new_price
+
+    def update_quantity(self, new_quantity, increment=True):
```

Comment:
- [Add InventoryItem class](https://github.com/username/repo/commit/abc123)
- [Add update_price method to InventoryItem class](https://github.com/username/repo/commit/def456)
- [Add update_quantity method to InventoryItem class](https://github.com/username/repo/commit/ghi789)

Hi @username,

Thanks for the review! I agree that unit tests would be beneficial for this function. I will add them in a separate commit for better tracking.

Regarding the function itself, I noticed that the `increment` parameter is optional but defaults to `True`. This might be confusing for users who expect the function to always increment the quantity. I suggest making `increment` a required parameter to make the function's behavior more predictable.

Here's an example of how the function signature could look like:
```python
def update_quantity(self, new_quantity: int, increment: bool = False):
```
This way, users who want to increment the quantity can still do so by passing `True` as the second argument, while those who want to set a specific quantity can pass the desired value as the first argument and `False` as the second argument.

I hope this helps! Let me know if you have any further suggestions or questions.

Best,
[Your Name]